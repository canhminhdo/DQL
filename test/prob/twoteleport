#!/bin/sh

DIR=../../test/prob
cd ../../examples/prob && maude < twoteleport.maude -no-banner -no-advise | sed -r '/^(rewrites:|Solution|states:)/d' > $DIR/twoteleport.out
diff $DIR/twoteleport.expected $DIR/twoteleport.out
rm $DIR/twoteleport.out