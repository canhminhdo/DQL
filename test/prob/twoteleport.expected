==========================================
reduce in TWO-TELEPORT : initQState |= [H(1) ; CX(1,2) ; H(3) ; CX(3,4) ; CX(0,
    1) ; H(0) ; CX(5,4) ; H(5) ; ((P(4,|0>) ? with prob == 1/2) ; skip U (P(4,
    |1>) ? with prob == 1/2) ; X(3)) ; ((P(5,|0>) ? with prob == 1/2) ; skip U
    (P(5,|1>) ? with prob == 1/2) ; Z(3)) ; ((P(1,|0>) ? with prob == 1/2) ;
    skip U (P(1,|1>) ? with prob == 1/2) ; X(2)) ; ((P(0,|0>) ? with prob ==
    1/2) ; skip U (P(0,|1>) ? with prob == 1/2) ; Z(2))]P(2,qubitAt(initQState,
    0)) /\ P(3,qubitAt(initQState, 5)) .
result JudgmentSet: emptyJS
Bye.
